src/test/java/duke/TaskListTest.java

```diff
@@ -0,0 +1,125 @@
+package duke;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.jupiter.api.Test;
+
+public class TaskListTest {
+    @Test
+    public void addTaskTest() {
+        TaskList list = new TaskList();
+        Task task;
+        String listString = "";
+        int size = 0;
+
+        // test 0
```

Comment:
Instead of putting several tests in one function, maybe you should split them up into several functions so it's easier to see which individual test breaks the code (now if this `addTaskTest` fails you will have to comb through all the mini tests here to see what went wrong).

Also I think it will be clearer if you added a short comment to describe what you are testing for in each test.